### DISCUSSION TITLE
Should the "working-directory" match the root directory defined in the Vercel project?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [amondnet/vercel-action#130] [Q&A] [answered]
This GitHub Action accept a setting called [working-directory](https://github.com/amondnet/vercel-action#inputs). It is not clear to me what is the purpose of it.

Is it the same as the root directory that we can set in the Vercel interface? If yes, should they match?

<img width="757" alt="Screen Shot 2022-03-08 at 1 20 52 PM" src="https://user-images.githubusercontent.com/8782666/157246022-0db1c8ed-2dd1-40f5-b365-85bbfc21f3d3.png">

**Comment by amondnet** [comment, marked as answer]
Using the `working-directory` input, you can specify the working directory of where to run this action. It has nothing to do with the root directory.

**Reply by PaulRBerg** [reply]
Great! Thanks for confirming.

**Reply by mesqueeb** [reply]
@amondnet @paulrberg to me, even with this answer it's still confusing what this means.

when you say:
> where to run this action

what does that mean exactly? If our working directory is a specific path of our monorepo, will only that part be uploaded? Otherwise the entire monorepo will be uploaded?
